src/guides/v2.3/frontend-dev-guide/templates/template-email-layout-handle.md

```diff
@@ -0,0 +1,57 @@
+---
+group: frontend-developer-guide
+title: Email templates layout handle
+functional_areas:
+  - Frontend
+---
+
+{:.bs-callout-warning}
+Magento 2.3.4 restricts the way that custom variables can be used within email templates.
```

Comment:
This would imply it was only 2.3.4 otherwise
```suggestion
Magento 2.3.4 and above restricts the way that custom variables can be used within email templates.
```